src/client/index.js

```diff
@@ -225,11 +225,13 @@ export const makeComponents = jedInstance => {
   Translate.propTypes = {
     children: PropTypes.any.isRequired,
     context: PropTypes.string,
+    comment: PropTypes.string,
```

Comment:
Ah I just realize that we pass on the rest params to the rendered element... so I guess there's some risk of reusing  `comment` here, at least when someone uses `as` with something that's not just a simple HTML element but some custom react element.

Maybe the name should be something like `translatorComment` or `i18nComment` that makes it much less likely to be used by any custom component?

Reply:
I changed it so that we always use comments, even inside JSX, and got rid of the prop:

```jsx
<div>
  {/* i18n: Title */}
  <Translate>Hello & World</Translate>
</div>
```